utils/mock-data-loader/bss/deals/deals-to-submit-to-tfm.js

```diff
@@ -0,0 +1,242 @@
+const { nowTimestamp } = require('../dates');
+const MANDATORY_CRITERIA = require('../mandatoryCriteria');
+const { MAKER } = require('../../portal/roles');
+
+// de-structure and create a new array, so `sort` doesn't impact 'MANDATORY_CRITERIA'
+let mandatoryCriteria = [...MANDATORY_CRITERIA].sort((a, b) => (a.version > b.version ? 1 : -1));
```

Comment:
Could we lodash deep clone if poss? This is a shallow clone -- not an issue here (yet!) but could later become one

Reply:
Also sortedMandatoryCriteria if poss

Reply:
General comment that we are sure this is a deal to submit in terms of params